Sources/ComposableArchitecture/Internal/Create.swift

```diff
@@ -20,25 +23,25 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import Combine
-import Darwin
+// import Darwin
 
 final class DemandBuffer<S: Subscriber>: @unchecked Sendable {
   private var buffer = [S.Input]()
   private let subscriber: S
   private var completion: Subscribers.Completion<S.Failure>?
   private var demandState = Demand()
-  private let lock: os_unfair_lock_t
+  private let lock: NSRecursiveLock
```

Comment:
This should probably switch off lock types depending on whether or not you can link Darwain